deploy/src/main/scala/com/typesafe/dbuild/deploy/Deploy.scala

```diff
@@ -278,13 +285,51 @@ class DeployHTTP(val log: Logger) extends IterativeDeploy[Unit] {
         if (out != relative) log.info("Deployed:  " + out)
       }
     } catch {
-       case e: NullPointerException => log.info("Deployed:  Response: " + response)
+       case e: NullPointerException => log.debug("Response: " + response)
+    }
+  }
+}
+
+// (pass to the constructor the deploy target uri as well)
+class DeployBintray(log: Logger, options: DeployInfo) extends DeployHTTP(log, options) {
+  override protected def init() = {
+    val target = new java.net.URI(options.uri)
+    val path = target.getPath
+    val parts = (if (path.head == '/') path.tail else path).split("/")
+    if (parts.length != 4) sys.error("In a Bintray deploy section, the path must contain four elements")
+    log.debug("Deploying to Bintray")
+    log.debug("owner  : " + parts(0))
+    log.debug("repo   : " + parts(1))
+    log.debug("package: " + parts(2))
+    log.debug("version: " + parts(3))
+    Option(target.getFragment) match {
+      case Some("release") => log.debug("The uri fragment is \"release\", so we will release after deploy.")
+      case Some(fragment) => log.debug("The uri fragment is \""+fragment+"\", so we will not release after deploy (must be \"release\").")
+      case None => log.debug("There is no fragment, so we will not release after deploy.")
+    }
+  }
+  private val bintrayBase = "https://api.bintray.com/content/"
```

Comment:
FYI - The URI of this is (was?) slightly different for Maven repositories, so we should make sure the docs specify this is going into a generic repo on bintray.